src/connections/destinations/catalog/actions-fullstory-cloud/index.md

```diff
@@ -13,9 +13,9 @@ versions:
 
 [FullStory](https://www.fullstory.com/){:target="_blank"} lets product and support teams easily understand everything about the customer experience. The Segment integration for FullStory helps accurately identify your customers within FullStory.
 
-FullStory’s cloud mode Segment integration allows you send user and event data to FullStory from your servers and Cloud Apps so that you apply it to your analysis throughout FullStory. For example, you could build a funnel to analyze drop-off of users who engaged with a certain marketing campaign.
+FullStory’s cloud mode Segment integration allows you enrich FullStory data by sending user properties and events from your servers and Cloud Apps so that you apply it to your analysis throughout FullStory. For example, you could build a funnel to analyze drop-off of users who engaged with a certain marketing campaign.
 
-If you want to use FullStory’s tagless autocapture, use the [FullStory Device Mode (Actions) web destination](/docs/connections/destinations/catalog/actions-fullstory/). However, if you want to capture custom user properties and events from other [server-side sources](/docs/connections/sources/#server) or [cloud apps](/docs/connections/sources/#cloud-apps), such as recurring subscription purchases, use this cloud mode destination.
+FullStory's cloud mode destination requires that you also use FullStory’s tagless autocapture, available through the [FullStory Device Mode (Actions) web destination](/docs/connections/destinations/catalog/actions-fullstory/). However, if you want enrich the autocapture data with custom user properties and events from other [server-side sources](/docs/connections/sources/#server) or [cloud apps](/docs/connections/sources/#cloud-apps), such as recurring subscription purchases, use this cloud mode destination.
```

Comment:
Looks like there's a small grammar issue in this text:

> However, if you want enrich the autocapture data [...]

Expecting:

> However, if you want to enrich the autocapture data [...]

Reply:
fixed!